### DISCUSSION TITLE
不懂就问，这个也会有请求过于频繁吗

### DISCUSSION THREAD

**Discussion by qiyezhishang** [mikumifa/biliTickerBuy#435] [提问专区]
看到大佬的cpp-buy公告说抢票间隔太短会报错，而且大佬你也推荐抢b站的票，请问这边会有抢的太快不让访问的问题吗？

**Comment by mikumifa** [comment]
不会，默认就好

**Reply by qiyezhishang** [reply]
OK，感谢大大！

**Comment by mikumifa** [comment]
https://github.com/mikumifa/biliTickerBuy/discussions/390
